tools/cppfind

```diff
@@ -2,6 +2,6 @@
 
 FUSE_CFLAGS="$(pkg-config --cflags fuse) -DFUSE_USE_VERSION=29"
 
-echo "#include <fuse.h>" | cpp ${FUSE_CFLAGS} | grep -q "${1}"
```

Comment:
Hmmm... okay. I'll see what I can do. Should possibly issue a bug report because that probably shouldn't happen.